src/main.rs

```diff
@@ -522,9 +522,15 @@ fn run() -> Result<(), failure::Error> {
                         .takes_value(true)
                 )
                 .arg(
-                    Arg::with_name("http")
-                    .help("Runs local server on http instead of https, only valid when running dev authenticated")
-                    .long("http")
+                    Arg::with_name("local-https")
+                    .help("Takes requests for local server via https instead of http")
```

Comment:
`--local-protocol http` or `--local-protocol https` -> need some parsing/validation to put in enum